Automattic/newspack-listings
# feat: remove borders and padding in editor to match front-end styles

### All Submissions:






### Changes proposed in this Pull Request:



Removes borders and padding from block editor elements so that styles more closely match both core WP design patterns and the front-end styles for listings.

Closes #9.

### How to test the changes in this Pull Request:

1. Check out this branch and run `npm run build`.
2. Create or edit a curated listing block in Specific Listings mode and add several listings.
3. Observe that the styles for rendered listing items more closely match the front-end. Example:

In the editor:
<img width="1662" alt="Screen Shot 2020-12-04 at 10 19 16 AM" src="https://user-images.githubusercontent.com/2230142/101193917-7da5ce80-361a-11eb-9a6b-99610d2b8028.png">

On the front-end:
<img width="1208" alt="Screen Shot 2020-12-04 at 10 22 04 AM" src="https://user-images.githubusercontent.com/2230142/101193983-94e4bc00-361a-11eb-86d5-73de6120e480.png">

4. Test that it's still possible/easy enough to select the parent Curated List wrapper block in order to edit block attributes, both through the block navigation UI and by clicking near/alongside each listing item.
5. Repeat steps 2-4 with a list in Query mode.

#### Bonus: Background color option

This branch also adds a new option for setting the background color of a list. To test:

1. Create a new list in Specific Listings mode and add several listings.
2. Open the Color Settings panel and observe a new color picker for Background Color.
3. Confirm that the background color of the list changes in both the editor and front-end when you select a background color. Also confirm that a `1em` padding is added to the list container if there's a background color applied.
4. Repeat steps 1-3 with a list in Query mode.

### Other information:
